test/callable-instance.js

```diff
@@ -0,0 +1,61 @@
+// @ts-nocheck
```

Comment:
Similar to above, tests should be type checked.

Reply:
Typescript was insisting that the ES5-style class should be written using ES6 classes, which contradicted the purpose of the test.  I will add @ts-ignore to the specific lines instead.